docs/gds-supported-platform/getting-started-gsp-local.md

```diff
@@ -148,109 +212,222 @@ You run an app by creating a [Kubernetes Deployment object](https://kubernetes.i
 
-## Connect to GOV.UK Prototype Kit
+    ```
+    kubectl get deployments prototype-kit-web
+    ```
+
+    You will see output similar to the following:
+
+    ```
+    NAME                               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    prototype-kit-web                  1         0         1            0           1m
+    ```
+    Once the `CURRENT` and `AVAILABLE` fields are set to `1`, the Deployment is live.
 
-1. Use `kubectl port-forward` to tunnel to the ingress Gateway:
+1. Run the following to check that the `service` is live:
+
+    ```
+    kubectl get service prototype-kit-web
+    ```
+
+    You will see output similar to the following:
+
+    ```
+    NAME                TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+    prototype-kit-web   ClusterIP   10.110.146.100   <none>        3000/TCP   1m
+    ```
+
+    Once the `CLUSTER-IP` and `PORT` fields are complete, the service is live.
+
+1. Run the following to check that the `virtualservice` is live:
+
+    ```
+    kubectl get virtualservice prototype-kit
+    ```
+    You will see output similar to the following:
+
+    ```
+    NAME            GATEWAYS                       HOSTS                                 AGE
+    prototype-kit   [gsp-gsp-cluster.gsp-system]   [prototype-kit.local.govsandbox.uk]   19h
+    ```
+    Once the `GATEWAYS` and `HOSTS` field have URLs in them, the `virtualservice` is live.
+
+Once the `service`, `virtualservice` and Deployment are live, you have successfully created a local GSP instance and deployed the `govuk-prototype-kit` app.
+
+You must now connect to the app.
+
+## Connect to the app
+
+By default, your app is not accessible outside of the local instance. To connect to this app, you must use [`port-forwarding`](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/).
+
+1. Run the following to connect to the app:
 
     ```
     sudo --preserve-env kubectl port-forward service/istio-ingressgateway -n istio-system 80:80
     ```
 
-1. Open `http://prototype-kit.local.govsandbox.uk` in your browser.
+    You will see output similar to the following example:
+
+    ```
+    sudo --preserve-env kubectl port-forward service/istio-ingressgateway -n istio-system 80:80
+    Password:
+    Forwarding from 127.0.0.1:80 -> 80
+    Forwarding from [::1]:80 -> 80
+    ```
+
+1. Open [http://prototype-kit.local.govsandbox.uk](http://prototype-kit.local.govsandbox.uk) in your browser.
+
+You have successfully deployed the `govuk-prototype-kit` app in your local GSP instance.
 
-You have successfully deployed the `govuk-prototype-kit` app in your local GSP cluster.
+## Destroy the local instance
 
-## Destroy cluster
+You should destroy your local GSP instance when you have finished testing your app.
 
-Run the following command to destroy the cluster:
+Run the following command to destroy the local instance:
 
 ```
 ./scripts/gsp-local.sh delete
```

Comment:
probably should include instructions on how to to remove the installed packages so that the users system is in the same state is was before the exercise
```
brew uninstall kubernetes-cli
brew uninstall kubernetes-helm
brew uninstall hyperkit
brew uninstall docker-machine-driver-hyperkit
brew cask uninstall minikube
```

Reply:
if you just run the script with delete you will still have the minikube and dependencies on the system

Reply:
agree